src/OpenTelemetry.Sampler.AWS/AWSXRayRemoteSamplerBuilder.cs

```diff
@@ -66,8 +67,17 @@ public AWSXRayRemoteSamplerBuilder SetEndpoint(string endpoint)
     /// <summary>
     /// Returns a <see cref="AWSXRayRemoteSampler"/> with configuration of this builder.
     /// </summary>
-    /// <returns>an instance of <see cref="AWSXRayRemoteSampler"/>.</returns>
-    public AWSXRayRemoteSampler Build()
+    /// <returns>an instance of <see cref="ParentBasedSampler"/>.</returns>
+    public ParentBasedSampler Build()
```

Comment:
```suggestion
    public Sampler Build()
```

Shouldn't it be just `Sampler` to avoid any future breaking changes?

Reply:
Good call. Updated the return type to be Trace.Sampler instead of ParentBasedSampler.